twitchpopups.htm

```diff
@@ -143,15 +146,34 @@ <h1 id="popuptext"></h1>
     
         
     // Animate off
-    function deleteAnimation() {
-        hotseatison = false;
+    const deleteAnimation = () => {
+        hotSeatIsOn = false;
         $("#popupbox").animate({width:0}, 500);
         $("#popuptext").animate({"opacity":0, "margin-left":"50px"}, 700);
     }
     
 
-    function onConnectedHandler(addr, port) {
+    function onConnectedHandler (addr, port) {
         console.log(`* Connected to ${addr}:${port}`);
     }
+
+    function onMessageDeletedHandler(channel, username, deletedMessage, userstate) {
+        if (username == hotSeatUser) {
+            deleteAnimation();
+        }
+    }
+
+    function onTimeoutHandler(channel, username, reason, duration, userstate) {
+        if (username == hotSeatUser) {
+            deleteAnimation();
+        }
+    }
+
+    function onBanHandler (channel, username, reason, userstate) {
+        if (username == hotSeatUser) {
+            deleteAnimation();
+        }
+    }

```

Comment:
- May as well convert these into arrow functions too. 
- The last three functions all do the same thing, why not just have one function? They also pass parameters that aren't used apart from username.
- Also the conditional should match the type too `===` instead of `==`

Reply:
This was more fun when it was Twitch stream focused and not doing it perfectly focused

Reply:
Sorry, I'm not trying to make it miserable, it's just my review. Nothing should be taken personally in a code review. We're all here to improve the code.

Feel free to ignore my suggestions if you like, I didn't request changes to be made, this was just a comment.